docs/elements.md

```diff
@@ -499,7 +499,21 @@ Fill in the blank field that allows for mathematical symbol input.
 
 **question.html**
 ```html
-<pl-symbolic-input answers-name="symbolic_math" variables="x, y" label="$z =$"></pl-symbolic-input>
+     # Enable the input: x + y + 1
```

Comment:
A bit of a nitpick here, but given that `#` is not valid HTML comment, this could lead people to copy-paste and see this in their questions instead. Maybe a mustache comment?
```suggestion
     {{! Enable the input: x + y + 1}}
```

same for the ones below.

Reply:
@jonatanschroeder 
Changed it to:
**question.html**
```html
     <!-- Enable the input: x + y + 1 -->
     <pl-symbolic-input answers-name="symbolic_math_example1" variables="x, y" label="$z =$" />
     
     <!-- Expect the input: log32(x**2) * zeta(5) + y -->
     <pl-symbolic-input answers-name="symbolic_math_example2" variables="x, y" 
          extra-functions="log32, zeta" label="$z =$" />
     
     <!-- Expect the input: ln((max(1-t, x) + min(y+2, z) - lg(w**2)) / 2) -->
     <pl-symbolic-input answers-name="symbolic_math_example3" variables="x, y, z, w, t" 
          enable-function-groups="calculus-extended & logarithmic & prog-extended" label="$z =$" />

     <!-- Expect the input: x * abs(cbrt_omega) * tebi / 137 + 1 - t^(3 / 2) -->
     <pl-symbolic-input answers-name="symbolic_math_example4" variables="x, t" 
          enable-function-groups="complex-variables" 
          enable-constant-groups="named-constants & named-units" label="$z =$" />
```

Reply:
I think it's preferable to use the Mustache comments here, e.g. `{{! ... }}` because by time students receive the generated code, the pl tag will have been constructed into something else, while the HTML-style tag will remain there literally. I admit it's nitpicking.

In particular, in the case where you wrote "_Expect_ the input..." if that is meant to be a note to staff about the answer, then you _definitely_ want to use the Mustache-style comments instead, so that student's can't find them.